diff --git a/flake.nix b/flake.nix
index 908610f..3380b45 100644
--- a/flake.nix
+++ b/flake.nix
@@ -79,7 +79,7 @@
             extraSpecialArgs = {
 
             };
-            pkgs = pkgsForSystem (args.system or "x86_64-linux");  # need to change for darwin, seems to be broken
+            pkgs = pkgsForSystem (args.system or "aarch64-darwin");  # need to change for darwin, seems to be broken
           }
           // {
             inherit (args) extraSpecialArgs;
